db/seeds.rb

```diff
@@ -16,8 +16,9 @@
 
 Park.create(
   name: "Arches National Park",
-  state: "Utah",
+  nps_page: "https://www.nps.gov/arch/index.htm",
```

Comment:
Since all these paths are the same would it make more sense just to save the `nps_key` as a field and build the url on the fly? 
```
class Park < blah
  def nps_page_email
    "https://www.nps.gov/#{nps_key}/index.htm"
  end
end
```

That way if the Park address url changes but keeps the same identifier you can change it once and let it work everywhere without having to change data